tests/test_network_model_data.py

```
@@ -182,63 +142,214 @@
+        elif 'FROM SurvivalCurve' in query:
+            return MagicMock(fetchall=lambda: [])
+        elif 'FROM TimePeriod' in query:
+            return MagicMock(fetchall=lambda: [(2020,), (2025,)])
+        elif 'FROM main.Commodity' in query:
+            return MagicMock(fetchall=lambda: [('s1',), ('p1',), ('d1',)])
+        elif 'FROM main.Commodity' in query:
+            return MagicMock(fetchall=lambda: [('s1',), ('p1',), ('d1',)])
+        elif "Commodity WHERE flag LIKE '%p%'" in query:
+            return MagicMock(fetchall=lambda: [('s1',), ('p1',), ('d1',)])
+        elif "Commodity WHERE flag LIKE '%w%'" in query:
+            return MagicMock(fetchall=lambda: [])
```

Ruff (0.14.1)

Prefer `list` over useless lambda

Replace with `lambda` with `list`

(PIE807)